config/machine_readable/register-to-vote.yml

```diff
@@ -1,57 +1,59 @@
   <p>You need to be on the electoral register to vote in elections or referendums.</p>
 
 faqs:
   - question: Who can register
     answer: >
-      <p>You can register if you’re both:</p>
+      <p>You must be aged 16 or over (or 14 or over in Scotland and Wales).</p>
+      <p>You must also be one of the following:</p>
       <ul>
-        <li>aged 16 or over (or 14 or over in Scotland)</li>
-        <li>a UK citizen (or an Irish, EU or Commonwealth citizen with a permanent UK address)</li>
+        <li>a British citizen</li>
+        <li>an Irish or EU citizen living in the UK</li>
+        <li>a Commonwealth citizen who has permission to enter or stay in the UK, or who does not need permission</li>
+        <li>a citizen of another country living in Scotland or Wales who has permission to enter or stay in the UK, or who does not need permission</li>
       </ul>
-      <p>You will not be able to vote until you’re 18. If you live in Scotland, you can vote in Scottish Parliament and local elections when you’re 16.</p>
+      <p>Check which <a href="/elections-in-the-uk?src=schema">elections you’re eligible to vote in</a>.</p>
 
   - question: Registering online
     answer: >
-      <p>Use this service to register to vote.</p>
-      <p>You only need to register once - not for every election.</p>
+      <p>It usually takes about 5 minutes.</p>
       <p><a rel="external" href="https://www.registertovote.service.gov.uk/register-to-vote/start?src=schema">Start now</a></p>
       <h2>What you need to know</h2>
       <p>You’ll be asked for your National Insurance number (but you can still register if you do not have one).</p>
+      <p>After you’ve registered, your name and address will appear on the electoral register.</p>
 
-  - question: How to check if you’re already registered
+  - question: Check if you’re already registered
     answer: >
-      <p>Contact your local Electoral Registration Office to <a href="/get-on-electoral-register?src=schema">find out if you’re already registered to vote</a>.</p>
+      <p><a href="/contact-electoral-registration-office?src=schema">Contact your local Electoral Registration Office</a> to find out if you’re already registered to vote.</p>
 
-  - question: Updating your registration
+  - question: Update your registration
     answer: >
       <p>You can also use the <a rel="external" href="https://www.registertovote.service.gov.uk/register-to-vote/start?src=schema">‘Register to vote’ service</a> to:</p>
       <ul>
         <li>change your name, address or nationality</li>
-        <li>get on or off the <a href="/electoral-register?src=schema">open register</a>
-      </li>
+        <li>get on or off the <a href="/electoral-register?src=schema">open register</a></li>
       </ul>
       <p>To do this, you need to register again with your new details (even if you’re already registered to vote).</p>
 
-  - question: Registering with a paper form
+  - question: Register with a paper form
```

Comment:
Can I double-check this, as the reference document says `Register using a paper form`

Reply:
Great spot, thank you!